src/components/main-panel/cardsData.ts

```diff
@@ -0,0 +1,213 @@
+          "With Armaan Malik, Mohit Chauhan, Shankar-Ehsaan-Loy and more",
+      },
+      {
+        image: "/images/radio/shreya-ghoshal.jpg",
+        title: "",
+        subtitle: "With A.R. Rahman, Atif Aslam, Vishal-Shekhar and more",
+      },
+      {
+        image: "/images/radio/sonu-nigam.jpg",
+        title: "",
+        subtitle: "With Pritam, Shaan, A.R. Rahman and more",
+      },
+    ],
+  },
+  {
+    heading: "Featured Charts",
+    imageBorder: "rounded-md",
+    data: [
+      {
+        image: "/images/charts/global.jpg",
+        title: "",
+        subtitle:
+          "Your weekly update of the most played tracks right now - Global.",
+      },
+      {
+        image: "/images/charts/top-songs-india.png",
+        title: "",
+        subtitle:
+          "Your weekly update of the most played tracks right now - India.",
+      },
+      {
+        image: "/images/charts/top-50-global.jpg",
+        title: "",
+        subtitle:
+          "Your daily update of the most played tracks right now - Global.",
+      },
+      {
+        image: "/images/charts/top-50-india.jpg",
+        title: "",
+        subtitle:
+          "Your daily update of the most played tracks right now - India.",
+      },
+      {
+        image: "/images/charts/viral-50.jpg",
+        title: "",
+        subtitle:
+          "Your daily update of the most viral tracks right now - Global.",
+      },
+      {
+        image: "/images/charts/viral-50-india.jpg",
+        title: "",
+        subtitle:
+          "Your daily update of the most viral tracks right now - India.",
+      },
+    ],
+  },
+  {
+    heading: "Playlists from our Editors",
+    imageBorder: "rounded-md",
+    data: [
+      {
+        image: "/images/playlist/dinner-chill.jpg",
+        title: "",
+        subtitle: "Gentle Indie songs for a relaxed dinner.",
+      },
+      {
+        image: "/images/playlist/dinner-jazz.jpg",
+        title: "",
+        subtitle:
+          "The gentle sound of some of the greatest voices and instrumentalists in Jazz. Your Jazz dinner soundtrack.",
+      },
+      {
+        image: "/images/playlist/kitchen-swagger.jpg",
+        title: "",
+        subtitle: "Gettin' figgy with it, bana-na-na-nanana",
+      },
+      {
+        image: "/images/playlist/feel-good.jpg",
+        title: "",
+        subtitle:
+          "An uplifting yet tasteful dinner playlist with a guaranteed feel good vibe.",
+      },
+      {
+        image: "/images/playlist/latin.jpg",
+        title: "",
+        subtitle:
+          "You bring the ingredients, we bring the flavor. Enjoy la cena...",
+      },
+      {
+        image: "/images/playlist/soul-cuisine.jpg",
+        title: "",
+        subtitle: "Perfect music for perfect food.",
+      },
+      {
+        image: "/images/playlist/dinner-friends.jpg",
+        title: "",
+        subtitle: "The perfect soundtrack to those long nights over dinner",
+      },
+    ],
+  },
+];
```

Comment:
why this file is in components directory ? 
if this is static move this to a directory called `data`

Reply:
Sure